probe-rs/src/probe/arm_jtag.rs

```diff
@@ -20,7 +20,7 @@ pub struct SwdSettings {
     /// When a WAIT response is received, the number of idle cycles
     /// will be increased automatically, so this number can be quite
     /// low.
-    pub num_idle_cycles_between_writes: usize,
```

Comment:
Is this never bigger than an u8? Generally if there is no actual size concerns we should always use usize over other types :)

Reply:
Makes sense, but I didn't want to immediately update every JTAGAccess implementation. I guess I will.